test/integration/improve_this_page_test.rb

```diff
@@ -17,4 +17,22 @@ class ImproveThisPageTest < ActionDispatch::IntegrationTest
       assert page.has_link?('Is there anything wrong with this page?', href: '/contact/govuk')
     end
   end
+
+  test "asks the user for feedback if there is something wrong with the page" do
+    using_javascript_driver do
+      setup_and_visit_example('service_manual_guide', 'service_manual_guide')
+
+      assert feedback_form_is_hidden
+
+      within('.improve-this-page') do
+        page.click_link 'No'
+      end
+
+      refute feedback_form_is_hidden
+    end
+  end
+
+  def feedback_form_is_hidden
+    page.find('.js-feedback-form')[:class].include?('js-hidden')
```

Comment:
I think Capybara already has methods to assert that an element is *actually* visible on the page, rather than tying this to a particular class?